ansys/dpf/core/elements.py

```diff
@@ -487,8 +487,8 @@ def element_types_field(self):
         >>> model = dpf.Model(examples.static_rst)
         >>> elements = model.metadata.meshed_region.elements
         >>> field = elements.element_types_field
-        >>> field.data
-        array([1, 1, 1, 1, 1, 1, 1, 1])
+        >>> print(field.data)
```

Comment:
It's preferable in these examples to use the `__repr__` rather than `print`.

Recommend revert.